inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -803,28 +771,50 @@
     end
 
     context "when there are no rules but the project exists" do
-      describe "GET /iam/v2beta/projects/:id/rules" do
-        it "returns an empty list" do
-          resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}/rules")
-          expect(resp.http_status).to eq 200
-          expect(resp.parsed_response_body[:rules]).to eq([])
-        end
+      before(:all) do
+        resp = automate_api_request("/apis/iam/v2beta/projects",
+         http_method: 'POST',
+         request_body: {
+           id: CUSTOM_PROJECT_ID,
+           name: "display name !#$#"
+         }.to_json
+       )
+       expect(resp.http_status).to eq 200
+     end
+
+     after(:all) do
+       resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}", http_method: 'DELETE')
+       expect(resp.http_status.to_s).to match(/200|404/)
+     end
+
+      it "GET /iam/v2beta/projects/:id/rules returns an empty list" do
+        resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}/rules")
+        expect(resp.http_status).to eq 200
+        expect(resp.parsed_response_body[:rules]).to eq([])
       end
 
-      describe "GET /iam/v2beta/rules" do
-        it "returns an empty list" do
-          resp = automate_api_request("/apis/iam/v2beta/rules")
-          expect(resp.http_status).to eq 200
-          expect(resp.parsed_response_body[:rules]).to eq([])
-        end
+      it "GET /iam/v2beta/rules returns an empty list" do
+        resp = automate_api_request("/apis/iam/v2beta/rules")
+        expect(resp.http_status).to eq 200
+        expect(resp.parsed_response_body[:rules]).to eq([])
       end
 
-      describe "GET /iam/v2beta/rules/:id" do
-        it "returns a 404" do
-          resp = automate_api_request("/apis/iam/v2beta/rules/not-found")
-          expect(resp.http_status).to eq 404
-          expect(resp.parsed_response_body[:rule]).to eq(nil)
-        end
+      it "GET /iam/v2beta/rules/:id returns a 404" do
```

Comment:
this is when the rule doesn't exist? or if we aren't authorized to view? (i'm guessing the former, but worth clarifying in the test name)